library/model/book.py

```diff
@@ -101,7 +108,31 @@ def __eq__(self, other):
         return NotImplemented
 
     def __str__(self):
-        return BookSerializer().serialize(self, "JSON")
+        return self.serialize(self, "JSON")
```

Comment:
We are not really sure about the benefits of removing the BookSerializer Class. We are not sure if it improves the readability.